tests/installation/addon_products_sle.pm

```diff
@@ -5,11 +5,40 @@ use testapi;
 
 sub run() {
     my $self = shift;
-    assert_screen 'inst-addon', 3;
-    send_key $cmd{"next"}, 1;    # done
 
-    if (check_screen("local-registration-servers", 10)) {
-        send_key $cmd{ok};
```

Comment:
As with my other comment in upgrade_select - is there any harm keeping this if check_screen?